package.json

```diff
@@ -8,7 +8,8 @@
     "start": "node app",
     "lint": "eslint .",
     "build": "tsc",
-    "test": "NODE_OPTIONS='--experimental-vm-modules --trace-warnings' jest",
+    "test": "NODE_ENV=test NODE_OPTIONS='--experimental-vm-modules --trace-warnings' jest  --coverage",
+    "test:e2e": "yarn test --config ./jest-e2e.config.js",
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏
```suggestion
    "test:e2e": "yarn test --config ./jest-e2e.config.json",
```